src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -47,6 +47,7 @@ final class ProgressBar
     private $overwrite = true;
     private $terminal;
     private $firstRun = true;
+    private $currentMessage = '';
```

Comment:
I'd suggest renaming the property to `previousMessage`, the make it default to `null`, and make it replace the `firstRun` propoerty, since it's not needed anymore (`null !== $this->previousMessage` would be the same as `$this->firstRun`, isn't it?)